pages/index.js

```diff
@@ -1,23 +1,16 @@
-import Head from "next/head";
 import Header from "../components/Header";
 import Footer from "../components/Footer";
-import Cards from "../components/Cards";
+import Page from "../components/Page";
+
 
-// Importing stylesheets
-import globalStyles from "../styles/global";
 
 export default function Home() {
-    return (
-        <div className="container">
-            <Head>
-                <title>Evolution</title>
-                <link rel="icon" href="/favicon.ico" />
-            </Head>
-            <Header />
-            <Footer />
-            <style jsx global>
-                {globalStyles}
-            </style>
-        </div>
-    );
+  return (
+    <div className="container">
```

Comment:
Don't need this outer container anymore. I missed it before I merged. I'll delete straight on master.